opencti-platform/opencti-graphql/src/modules/ingestion/ingestion-taxii.ts

```diff
@@ -49,6 +49,8 @@ const INGESTION_DEFINITION: ModuleDefinition<StoreEntityIngestionTaxii, StixInge
     { name: 'added_after_start', label: 'Added after', type: 'date', mandatoryType: 'external', editDefault: true, multiple: false, upsert: true, isFilterable: true },
     { name: 'current_state_cursor', label: 'Current state cursor', type: 'string', format: 'short', mandatoryType: 'no', editDefault: true, multiple: false, upsert: true, isFilterable: true },
     { name: 'ingestion_running', label: 'Running', type: 'boolean', mandatoryType: 'external', editDefault: true, multiple: false, upsert: true, isFilterable: true },
+    { name: 'taxii_more', label: 'Taxii response more value', type: 'string', format: 'short', mandatoryType: 'no', editDefault: true, multiple: false, upsert: true, isFilterable: true },
+    { name: 'ingestion_last_run', label: 'Last ingestion run', type: 'date', mandatoryType: 'no', editDefault: true, multiple: false, upsert: true, isFilterable: true },
```

Comment:
I suggest to rename this field to "last_execution_date" since we already have this attribute for other types (background tasks & retention rules), and it's already mapped in the database.
We could only rename the field name, and keep the label if we want to have "last ingestion run".